modules/nw-networkpolicy-deny-all-allowed.adoc

```diff
@@ -37,55 +38,58 @@ endif::microshift[]
 apiVersion: k8s.cni.cncf.io/v1beta1
 kind: MultiNetworkPolicy
 metadata:
   name: deny-by-default
-  namespace: my-project <1>
+  namespace: my-project
   annotations:
-    k8s.v1.cni.cncf.io/policy-for:<namespace_name>/<network_name> <2>
+    k8s.v1.cni.cncf.io/policy-for:<namespace_name>/<network_name>
 spec:
-  podSelector: {} <3>
-  policyTypes: <4>
-  - Ingress <5>
-  ingress: [] <6>
+  podSelector: {}
+  policyTypes:
+  - Ingress
+  ingress: []
+----
++
+where:
++
+`namespace`:: Specifies the namespace in which to deploy the policy. For example, the `my-project` namespace.
+`annotations`:: Specifies the name of namespace project followed by the network attachment definition name.
+`podSelector`:: If this field is empty, the configuration matches all the pods. Therefore, the policy applies to all pods in the `my-project` namespace.
+`policyTypes`:: Specifies a list of rule types that the `NetworkPolicy` relates to.
+`- Ingress`:: Specifies `Ingress` only `policyTypes`.
+`ingress`:: Specifies ingress rules. If not specified, all incoming traffic is dropped to all pods.
 endif::multi[]
 ifndef::multi[]
+[source,yaml]
+----
 kind: NetworkPolicy
 apiVersion: networking.k8s.io/v1
 metadata:
   name: deny-by-default
-  namespace: my-project <1>
+  namespace: my-project
 spec:
-  podSelector: {} <2>
-  ingress: [] <3>
-endif::multi[]
+  podSelector: {}
+  ingress: []
 ----
-ifdef::multi[]
-<1> Specifies the namespace in which to deploy the policy. For example, the `my-project` namespace.
-<2> Specifies the name of namespace project followed by the network attachment definition name.
-<3> If this field is empty, the configuration matches all the pods. Therefore, the policy applies to all pods in the `my-project` namespace.
-<4> Specifies a list of rule types that the `NetworkPolicy` relates to.
-<5> Specifies `Ingress` only `policyTypes`.
-<6> Specifies `ingress` rules. If not specified, all incoming traffic is dropped to all pods.
-endif::multi[]
-ifndef::multi[]
-<1> `Specifies the namespace in which to deploy the policy. For example, the `my-project` namespace.
-<2> If this field is empty, the configuration matches all the pods. Therefore, the policy applies to all pods in the `my-project` namespace.
-<3> There are no `ingress` rules specified. This causes incoming traffic to be dropped to all pods.
-endif::multi[]
 +
+where:
++
+`namespace`:: Specifies the namespace in which to deploy the policy. For example, the `my-project` namespace.
+`podSelector`:: If this field is empty, the configuration matches all the pods. Therefore, the policy applies to all pods in the `my-project` namespace.
+`ingress`:: Where `[]` indicates that no `ingress` rules are specified. This causes incoming traffic to be dropped to all pods.
+endif::multi[]
+
 . Apply the policy by entering the following command. Successful output lists the name of the policy object and the `created` status.
 +
 [source,terminal]
 ----
 $ oc apply -f deny-by-default.yaml
 ----
-+
```

Comment:
we still want this note yeah?

Reply:
No, as it repeats what's already in the lead-in sentence. I found a few of these repeated comments.